doc/ug/dv_methodology.md

```diff
@@ -299,65 +299,124 @@ Once the regression tool is developed, it will provide a way to enable these cap
-They provide us the ability to gauge to what extent is our stimulus through constrained random tests covering the test intent, for all tests in the testplan.
-
-These constructs are typically encapsulated in a class and are sometimes referred to as 'functional coverage model'.
-They are sampled in 'reactive' components of the testbench, such as monitors and / or the scoreboards.
+These are more complex constructs that capture whether signals (that reflect the current state of the design) have met a set of values or a sequence of values that are interesting / corner cases.
+These constructs also allow us to capture whether multiple scenarios have occurred simultaneously through crosses.
+This is also often referred to as cross coverage.
+These constructs are typically encapsulated in a class and are sometimes referred to as the ‘functional coverage model’.
+They are sampled in ‘reactive’ components of the testbench, such as monitors and/or the scoreboards.
 They can also be embedded within the RTL to sample interesting scenarios through DV stimulus.
 
 Here are the metrics used with a brief explanation:
 
-*  **Assert Coverage**: This metric measures which assertions, cover properties and sequences have been exercised in the course of the simulation.
+* **Assert Coverage**: This metric measures which assertions, cover properties, and sequences have been exercised in the course of the simulation.
   Note, an assertion precondition counts as a cover point.
-*  **Covergroup Coverage**: This metric (sometimes called "Testbench Coverage") measures covergroups during simulation.
-  These are usually coded in the testbench to check that the desired stimulus was generated properly, but can also be embedded in the RTL.
-
-Code coverage can reach 100% but it may not be sufficient to indicate whether all interesting combinations of scenarios were exercised (such as boundary conditions, concurrent scenarios and different CSR fields holding specific combinations of values resulting in the DUT being exercised in interesting ways).
-Hence, developing a detailed functional coverage model in conjunction with constrained random test sequences is key to high quality DV effort.
-
-Functional coverage can also reach 100% while the code coverage may not.
-This would indicate that the testplan is either incomplete or there are undocumented or unreachable features in the design.
-As such they can be addressed accordingly through either updating the design specification and augmenting the testplan / written tests, or by optimizing the design to remove unreachable logic if possible.
-There may be features that cannot be tested and cannot be removed from the design either.
-These would have to be analyzed and excluded from the coverage collection as a waiver.
-This is an exercise the DV and the designer typically perform together.
-This is discussed in more detail below.
```

Comment:
This section provides a nice segue into the next topic of exclusions. How about we retain this?

Reply:
@sriyerg  I worked this into the paragraph starting on line 409
is this ok?